src/com/muntian/ht46_post/synchro_post/Post.java

```diff
@@ -0,0 +1,54 @@
+package com.muntian.ht46_post.synchro_post;
+
+public class Post {
+    private int _parcelsLimit;
+    private int _parcelsNumber;
+    private int _acceptedParcelsNumber;
+    private int _sentParcelsNumber;
+
+    public Post(int parcelsLimit) {
+        _parcelsLimit = parcelsLimit;
+    }
+
+    public int get_parcelsLimit() {
```

Comment:
This method and methods below do not follow Java Code Convention. Please rename from get_parcelsLimit to getParcelsLimit and so on.